app/helpers/api/bapi/getCatalogueItemPricing.js

```diff
@@ -0,0 +1,16 @@
+import { getData } from 'buying-catalogue-library';
+import { getEndpoint } from '../../../endpoints';
+import { logger } from '../../../logger';
+
+export const getCatalogueItemPricing = async ({ accessToken, catalogueItemId, loggerText = 'Catalogue item' }) => {
+  const cataloguePricingEndpoint = getEndpoint({ api: 'bapi', endpointLocator: 'getCatalogueItemPricing', options: { catalogueItemId } });
```

Comment:
as bit of a rule we are moving away from calling the getEndpoint anymore and moving that within here. see this as bit of an example

```
const getOrderItemEndpoint = (orderId, orderItemId) => (
  `${orderApiUrl}/api/v1/orders/${orderId}/order-items/${orderItemId}`
);

export const getOrderItem = async ({ orderId, orderItemId, accessToken }) => {
  const endpoint = getOrderItemEndpoint(orderId, orderItemId);
  const catalogueOrderItem = await getData({ endpoint, accessToken, logger });
  logger.info(`Catalogue order item returned for ${orderItemId}`);

  return catalogueOrderItem;
};
```

also remove `getCatalogueItemPricing` from the endpoints map